JASP-Engine/JASP/po/R-nl.po

```diff
@@ -754,17 +754,19 @@ msgstr ""
 msgid "<i>This information allows for calculation of:</i>"
 msgstr "<i>Met deze informatie worden de volgende zaken berekend:</i>"
 
-msgid "The point estimate of the true population value %1$s"
-msgstr "De puntschatting van de ware populatiewaarde %1$s"
+msgid ""
+"The point estimate of the true population value <i>W&#770 = N × w&#772</i>"
+msgstr ""
+"De puntschatting van de ware populatiewaarde <i>W&#770 = N × w&#772</i>"
 
 msgid "The uncertainty of the estimator %1$s%2$s%3$s"
 msgstr "De onzekerheid van de schatter %1$s%2$s%3$s"
 
-msgid "The confidence interval around the point estimate %1$s"
-msgstr "Het betrouwbaarheidsinterval rond de puntschatting %1$s"
+msgid "The confidence interval around the point estimate <i>W&#770 ± U</i>"
+msgstr "Het betrouwbaarheidsinterval rond de puntschatting <i>W&#770 ± U</i>"
```

Comment:
Are these correct?

Reply:
The confidence interval around the point estimate Ŵ ± U
Het betrouwbaarheidsinterval rond de puntschatting Ŵ ± U

The point estimate of the true population value Ŵ = N × w̄
De puntschatting van de ware populatiewaarde Ŵ = N × w̄
 
I do see the italic in the real result:

![Screen Shot 2020-05-20 at 09 45 57](https://user-images.githubusercontent.com/15656960/82419481-c7fc9f00-9a7e-11ea-8634-fb6ce4ee81e6.png)

So I think it is correct

Reply:
Might be good to change the html characters to unicode too
`W&#770` is `\U0174` for example